stories/DataTable.js

```diff
@@ -1,68 +1,156 @@
+      flavor="fixed-layout"
+      onSelect={action()}
     >
-      <thead>
-        <Row
-          head={boolean('Row 1 is Head', true)}
-          variation={array('Row 1 variation', []) || undefined}
-        >
-          <Cell scope="col" truncate>Opportunity Name</Cell>
-          <Cell scope="col" truncate>Account Name</Cell>
-          <Cell scope="col" truncate>Close Date</Cell>
-          <Cell scope="col" truncate>Stage</Cell>
-          <Cell scope="col" truncate>Confidence</Cell>
-          <Cell scope="col" truncate>Amount</Cell>
-          <Cell scope="col" truncate>Contact</Cell>
-        </Row>
-      </thead>
-      <tbody>
-        <Row variation={array('Row 2 variation', ['hint-parent']) || undefined}>
-          <Cell truncate data-label="Opportunity Name" scope="row"><a href="#top">Cloudhub</a></Cell>
-          <Cell truncate data-label="Account Name">Cloudhub</Cell>
-          <Cell truncate data-label="Close Date">4/14/2015</Cell>
-          <Cell truncate data-label="Stage">Prospecting</Cell>
-          <Cell truncate data-label="Confidence">20%</Cell>
-          <Cell truncate data-label="Amount">$25k</Cell>
-          <Cell truncate data-label="Contact">jrogers@cloudhub.com</Cell>
-        </Row>
-        <Row variation={array('Row 3 variation', []) || undefined}>
-          <Cell
-            variation={array('Cell 3-1 variation', ['cell-shrink']) || undefined}
-            truncate
-            data-label="Opportunity Name"
-            scope="row"
-          ><a href="#top">
-          Cloudhub + Anypoint Connectors + Some other guys with a long name</a>
-          </Cell>
-          <Cell truncate data-label="Account Name">Cloudhub</Cell>
-          <Cell truncate data-label="Close Date">4/14/2015</Cell>
-          <Cell truncate data-label="Stage">Prospecting</Cell>
-          <Cell truncate data-label="Confidence">20%</Cell>
-          <Cell truncate data-label="Amount">$25k</Cell>
-          <Cell truncate data-label="Contact">jrogers@cloudhub.com</Cell>
-        </Row>
-        <Row variation={array('Row 4 variation', []) || undefined}>
-          <Cell truncate data-label="Opportunity Name" scope="row"><a href="#top">
-          Mr. B</a>
-          </Cell>
-          <Cell truncate data-label="Account Name">B-Man</Cell>
-          <Cell truncate data-label="Close Date">5/14/2015</Cell>
-          <Cell truncate data-label="Stage">Winning</Cell>
-          <Cell truncate data-label="Confidence">&gt;99%</Cell>
-          <Cell truncate data-label="Amount">$25k</Cell>
-          <Cell truncate data-label="Contact">jrogers@cloudhub.com</Cell>
-        </Row>
-      </tbody>
-    </Table>
+      <DataTableColumn
+        dataKey="col1"
+        sortable
+        title="Column 1"
+      />
+      <DataTableColumn
+        dataKey="col3"
+        sortable
+        title="Column 3"
+      />
+      <DataTableActionColumn
+        dataKey="stuff"
+        cellRenderer={({ dataKey, rowIndex, defaultProps }) => (
```

Comment:
There is a `defaultActionHeadRenderer`. Shouldn't this use it? Looks a bit broken:

<img width="956" alt="screen shot 2017-12-08 at 11 02 56" src="https://user-images.githubusercontent.com/1682504/33760820-6435356e-dc07-11e7-900a-8c9e2d9384c1.png">

Reply:
It does (it's the default in `DataTableActionColumn.defaultProps`). What browser is that? Looks normal for me in FF/Chrome.

Reply:
FF, it renders:

``` html
<div class="slds-th__action">
<span class="slds-assistive-text">Actions</span>
</div>
```

which explains the absence of visible title